cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -154,6 +175,13 @@ def _cleanup():
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
 
+    rhv_prov = v2v_providers[1]
+    collection = rhv_prov.appliance.provider_based_collection(rhv_prov)
+    # validate MAC address matches between source and target VMs
+    for src_vm in form_data_dual_vm_obj_dual_datastore[1]:
+        migrated_vm = collection.instantiate(src_vm.name, rhv_prov)
+        soft_assert(src_vm.mac_address == migrated_vm.mac_address)
```

Comment:
Same comment as above. Also see soft_assert being used here, is this because you want to go through all items?

Reply:
yes, we want to go through all the items. Don't want to fail if one failed. hence soft_assert.